mappings/net/minecraft/world/entity/SectionedEntityCache.mapping

```diff
@@ -0,0 +1,45 @@
+CLASS net/minecraft/class_5573 net/minecraft/world/entity/SectionedEntityCache
+	COMMENT Stores entities with the chunk sections they are in.
+	FIELD field_27250 entityClass Ljava/lang/Class;
+	FIELD field_27251 posToStatus Lit/unimi/dsi/fastutil/longs/Long2ObjectFunction;
+	FIELD field_27252 chunkSections Lit/unimi/dsi/fastutil/longs/Long2ObjectMap;
+	FIELD field_27253 trackedSections Lit/unimi/dsi/fastutil/longs/LongSortedSet;
+	METHOD <init> (Ljava/lang/Class;Lit/unimi/dsi/fastutil/longs/Long2ObjectFunction;)V
+		ARG 1 entityClass
+		ARG 2 chunkStatusDiscriminator
+	METHOD method_31770 getChunkPositions ()Lit/unimi/dsi/fastutil/longs/LongSet;
+	METHOD method_31771 getSections (II)Lit/unimi/dsi/fastutil/longs/LongSortedSet;
+		ARG 1 chunkX
+		ARG 2 chunkZ
+	METHOD method_31772 getSections (J)Ljava/util/stream/LongStream;
+		ARG 1 chunkPos
+	METHOD method_31773 forEachIntersects (Lnet/minecraft/class_5575;Lnet/minecraft/class_238;Ljava/util/function/Consumer;)V
+		ARG 1 filter
+		ARG 2 box
+		ARG 3 action
+	METHOD method_31775 intersecting (Lnet/minecraft/class_238;)Ljava/util/function/Predicate;
+		ARG 0 box
+	METHOD method_31777 forEachInBox (Lnet/minecraft/class_238;Ljava/util/function/Consumer;)V
```

Comment:
What's the difference between `InBox` and `Intersects`?

Reply:
`Insersects` uses `Box#intersects` and is filtered from `InBox` which does its own checking logic

Reply:
But do they not perform the same action? Also there is still the issue that the functions do not clarify whether they are working with entities or groups/sections

Reply:
One executes a callback, one returns a predicate .. they are nothing alike.

Reply:
`forEachIntersecting` would be acceptable